### PR TITLE
Updated PortX.app v1.1.5

### PR DESCRIPTION
**Pull request by netsarang-bigseungho**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [ ] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/portx.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask "portx" do
-  version "1.1.4,10.2"
+  version "1.1.5,8"
   sha256 :no_check
```
Comment (SMillerDev) [inline Casks/portx.rb:3]: This can be re-enabled then.

Comment (NetSarangComputer) [reply Casks/portx.rb:3]: What do you mean be re-enable?

Comment (SMillerDev) [reply Casks/portx.rb:3]:
```suggestion
  sha256 "SOME_SHA"
```

Comment (NetSarangComputer) [reply Casks/portx.rb]: Thank you for your offer.

**Pull request merged**